Sources/ComposableArchitecture/ViewStore.swift

```diff
@@ -199,7 +199,7 @@ public final class ViewStore<State, Action>: ObservableObject {
   ///
   /// For example, an alert binding can be dealt with like this:
   ///
-  ///     struct State { var alert: String? }
+  ///     typealias State = String
```

Comment:
Ha I'm a lil confused about these examples in general now. Since we have `alert.message` below...

Reply:
Yeah this one has more things wrong with it than I saw at first. This probably got out of sync after the alert stuff.